src/index.js

```diff
@@ -9,10 +9,10 @@ import rootReducer from "./redux/reducers/_rootReducer";
 const store = createStore(rootReducer);
 
 ReactDOM.render(
-  <React.StrictMode>
-    <Provider store={store}>
-      <App />
-    </Provider>
-  </React.StrictMode>,
+  // <React.StrictMode>
```

Comment:
Still need to contact a TA about not using StrictMode. Sorry, I'll do that tonight.